config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,20 @@
+# If PMP_OK_URL set, then OkComputer will mount a route at that location, otherwise it is effectively disabled
+OkComputer.mount_at = ENV["PMP_OK_URL"] || false
+
+# For built-in checks, see https://github.com/sportngin/okcomputer/tree/master/lib/ok_computer/built_in_checks
+redis_url = ENV['PLUM_REDIS_URL'] || 'localhost'
+redis_port = ENV['PLUM_REDIS_PORT'] || '6379'
+OkComputer::Registry.register "redis", OkComputer::RedisCheck.new(host: redis_url, port: redis_port)
+
+iiif_url = ENV["PMP_IIIF_URL"] || "http://localhost/imagesrv"
+OkComputer::Registry.register "iiif", OkComputer::HttpCheck.new(iiif_url)
+
+solr_url = ENV["PMP_SOLR_URL"] || 'http://localhost:8983/solr/hydra-development'
```

Comment:
Files in `initializers` are run in alphabetical order, but maybe we can make use of the after_initialize hook? http://guides.rubyonrails.org/configuring.html#initialization-events